### PR TITLE
feat(post): Creat Swift App Post

### PR DESCRIPTION
**Pull request by KRMeserve**
Adding my post to the blog!

## Screenshot:
Screen Shot 2020-09-26 at 9 29 16 AM

### CODE REVIEW

**Comment by KRMeserve** [comment]
I'm not entirely sure why there wasn't a `yarn-lock` file before since the readme has us use yarn. Let me know if I shouldn't commit the yarn lock.

File: src/content/swift-app/index.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    if (currentHiScore as? Int) != nil {
+      setHighScoreText(number: currentHiScore as! Int)
+    } else {
+      setHighScoreText(number: 0)
+    }
+
+  }
+```
+
+The ViewController should come pre-built with a viewDidLoad lifecycle function. This function is run once the app loads to populate the view and give functionality to the elements. This function is called after the view controller has loaded its view hierarchy into memory. You usually override this method to perform additional initialization on views that were loaded.
```
Comment (trybick) [inline src/content/swift-app/index.md]: Consider putting viewDidLoad in backticks ``

File: src/content/swift-app/index.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+This Storyboard/Controller setup creates a coding environment that allows you to look at what the app will look like in the Main.Storyboard file and practically drag and drop buttons, images, or any DOM element into your ViewController to create variables! After your UI is connected to a function or variable, a helpful icon by the line number will highlight that element on your storyboard.
+
+![](./connecting-button-to-function-swift.png)
+
+You can add elements directly to the Main.Storyboard, as well. Positioning elements is as easy as dragging and dropping them within the storyboard. To make the app responsive across different iOS screen sizes, Xcode also provides the ability to add responsive constraints on each element to control their size and positioning.
```
Comment (trybick) [inline src/content/swift-app/index.md]: Consider putting Main.Storyboard in backticks ``

**Review by trybick — APPROVED** [review]
Awesome! Just two small nits, but not blocking. Looks really good.

Committing yarn.lock is a good idea. Do you mind to delete the package-lock.json file too? I probably created it by mistake.

**Pull request merged**